src/pages/project/project_wizard/ProjectDescriptionForm.tsx

```diff
@@ -16,6 +22,9 @@ interface ProjectDescriptionFormProps {
 export const ProjectDescriptionForm = ({ thisIsRekProject }: ProjectDescriptionFormProps) => {
   const { t } = useTranslation();
   const { values, setFieldValue, setFieldTouched } = useFormikContext<SaveCristinProject>();
+  const [title, setTitle] = useState(values.title);
+  const debouncedTitle = useDebounce(title);
```

Comment:
Egen `useState` for dette ser ut som bare dobbeltlagring av `values.title` 🤔 
Blir ikke dette det samme bare med mindre kode og færre rerenders?

```suggestion
  const debouncedTitle = useDebounce(values.title);
```